toolkit/tools/pkg/imagecustomizerlib/bootcustomizer.go

```diff
@@ -88,13 +94,23 @@ func (b *BootCustomizer) getSELinuxModeFromGrub() (imagecustomizerapi.SELinuxMod
 
 	// Get the SELinux kernel command-line args.
 	if b.isGrubMkconfig {
+		// If grub config file is missing, indicate that with fs.ErrNotExist.
+		if b.defaultGrubFileContent == "" {
+			return imagecustomizerapi.SELinuxModeDefault, fs.ErrNotExist
```

Comment:
Let's stop using `fs.ErrNotExist` to represent when something is missing. It just causes problems. Add a `bool` to the return list instead.